modules/ballerina-core/src/test/resources/lang/connectors/connector-test.bal

```diff
@@ -0,0 +1,24 @@
+connector Foo (string name, int age) {
+    int userAge = 10 + age;
+    string userName = "Ballerina" + name;
+
+    action getUserName () (string) {
+        return userName;
+    }
+
+    action getAge () (int) {
+        return userAge;
+    }
+}
+
+function getFooConnector(string name, int age) (Foo) {
+    Foo foo = create Foo(name, age);
+    return foo;
+}
+
+function testConnectorReturn (string name, int age) (string) {
+    Foo conn = getFooConnector(name, age);
+    string userName = conn.getUserName();
+    int userAge = conn.getAge();
+    return "Username: " + userName + ", Age: " + userAge;
+}
```

Comment:
We need to add test case which is taking an connector as argument to function as well. Refer the [discussion](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/ballerina-dev/uasy0zaPEZI/IturA6iMCQAJ)